core-bundle/tests/EventListener/DataContainer/ValidateCustomRgxpListenerTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Tests\EventListener\DataContainer;
+
+use Contao\CoreBundle\EventListener\DataContainer\ValidateCustomRgxpListener;
+use Contao\CoreBundle\Tests\TestCase;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class ValidateCustomRgxpListenerTest extends TestCase
+{
+    public function testThrowsExceptionIfInvalidRegex(): void
+    {
+        $this->expectException(\Exception::class);
+        $this->expectExceptionMessage('ERR.invalidCustomRgxp');
```

Comment:
This should go into line 34, so we do not accidentally catch a prior exception. Also, the class needs to be adjusted to `\InvalidArgumentException::class`.